### DISCUSSION TITLE
I am unable to deploy the fund function

### DISCUSSION THREAD

**Discussion by freeman237** [Cyfrin/foundry-full-course-cu#2340] [Q&A]
![Uploading kk.png…]()

**Comment by EngrPips** [comment]
Hello @freeman237, The image you send isn't loading.

**Reply by freeman237** [reply]
![kk](https://github.com/user-attachments/assets/779b6e9e-54a9-4db5-9555-b62c092281c5)

**Reply by EngrPips** [reply]
Try changing the evm version to `cancum` and then retry.

**Comment by AboloreDev** [comment]
Can't seem to see a problem with the code, maybe there is an issue with your deployment environment. Try changing the environment to cancun and retry the deployment
